dprint.json

```diff
@@ -1,11 +1,14 @@
 {
-  "$schema": "https://dprint.dev/schemas/v0.json",
-  "incremental": true,
```

Comment:
why did you decide to remove it again?

Reply:
I thought it was not needed and was breaking but I'll add it again